src/unit/test_zmalloc.c

```diff
@@ -0,0 +1,56 @@
+#include "../zmalloc.c"
+#undef UNUSED
+#include "test_help.h"
+
+int test_zmallocInitialUsedMemory(int argc, char **argv, int flags) {
+    UNUSED(argc);
+    UNUSED(argv);
+    UNUSED(flags);
+
+    printf("Malloc prefix size: %d\n", (int) PREFIX_SIZE);
```

Comment:
```suggestion
    TEST_PRINT_INFO("Malloc prefix size: %d\n", (int) PREFIX_SIZE);
```
I think we want to in the future suppress the output from passing tests, so would rather wrap it in the test print function.